examples/usart_dma_rx_circular/usart_dma_rx_circular.c

```diff
@@ -0,0 +1,84 @@
+#include "ch32fun.h"
```

Comment:
It would be prefeable to have the commend here explaining it as well.  You could copy-paste and drop README.md or not drop it.  Up to you.